mail_gateway/wizards/mail_compose_gateway_message.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MailComposeGatewayMessage(models.TransientModel):
+    _name = "mail.compose.gateway.message"
+    _inherit = "mail.compose.message"
```

Comment:
You need to deal with m2m relations:


TypeError: Many2many fields mail.compose.gateway.message.object_attachment_ids and mail.compose.message.object_attachment_ids use the same table and columns